### PR TITLE
dist/common/scripts: generate debug log when exception occurred

### PR DESCRIPTION
**Pull request by syuu1228**
Using traceback_with_variables module, generate more detail traceback
with variables into debug log.
This will help fixing bugs which is hard to reproduce.

### CODE REVIEW

**Comment by syuu1228** [comment]
Note:
Why it saves the log into /var/tmp is, because some script like scylla_prepare will launch in systemd, which will run in PWD=/, so it should save somewhere else.
I guess /var/tmp is suitable for this purpose, but we also can consider to another place such as:
 - home directory of current user. but almost all script will run in root, it will be /root
 - /var/lib/scylla
 - /var/log

**Comment by syuu1228** [comment]
Since perftune.py is part of Seastar, perftune.py part is posted at seastar-ml:
https://groups.google.com/g/seastar-dev/c/PxyeksYThAk

File: install-dependencies.sh [hunk 1]
```diff
@@ -293,6 +294,7 @@ elif [ "$ID" = "fedora" ]; then
     pip3 install "geomet<0.3,>=0.1"
     # Disable C extensions
     pip3 install scylla-driver --install-option="--no-murmur3" --install-option="--no-libev" --install-option="--no-cython"
+    pip3 install traceback-with-variables
```
Comment (avikivity) [inline install-dependencies.sh:297]: Please check that the relocatable package correctly finds the module. I don't know if it looks for pip modules when it packages python modules.

Comment (syuu1228) [reply install-dependencies.sh:297]: @avikivity I tested the module worked on deb/rpm packaged relocatable scylla-python3, by adding "raise Exception" on scripts, it just worked.
Also I verified traceback-with-variables PIP package is added at /opt/scylladb/python3/lib64/python3.9/site-packages/.

**Pull request closed without merging**